infrastructure/templates/template.json

```diff
@@ -997,16 +997,54 @@
         "criterionType": "StaticThresholdCriterion"
       },
       {
-        "name": "[concat(parameters('subscription'),'DataFactoryPipelineFailures')]",
-        "description": "Data Factory - Pipeline Failed Run",
-        "resourceId": "[resourceId('Microsoft.DataFactory/factories', variables('dataFactoryName'))]",
-        "resourceType": "Microsoft.DataFactory/factories",
-        "metricName": "PipelineFailedRuns",
-        "threshold": 1,
-        "operator": "GreaterThan",
-        "timeAggregation": "Total",
-        "criterionType": "StaticThresholdCriterion"
-      }
+        "type": "microsoft.insights/metricAlerts",
+        "apiVersion": "2018-03-01",
+        "name": "[parameters('metricAlerts_Data_Factory_failed_pipeline_runs_name')]",
+        "location": "global",
+        "dependsOn": [
+            "[resourceId('microsoft.insights/actionGroups', parameters('actionGroups_s101p01_ag_ees_alertedusers_name'))]"
+        ],
+        "tags": {
+            "Environment": "Prod",
+            "Parent Business": "Business Intelligence",
+            "Portfolio": "Data Directorate",
+            "Product": "Enterprise Data and Analytics Platform (EDAP)",
+            "Service Line": "Data Operations",
+            "Service Offering": "Enterprise Data and Analytics Platform (EDAP)",
+            "Service": "Business Intelligence"
+        },
+        "properties": {
+            "severity": 1,
+            "enabled": true,
+            "scopes": [
+                "[parameters('factories_s101p01_df_ees_release_externalid')]"
```

Comment:
I think this might be our sticking point with making this change - I don't think we have access to data factory details in this template - but I'm not sure